hedera-node/hedera-networkadmin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/config/NetworkAdminServiceConfig.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.networkadmin.impl.config;
+
+import com.hedera.node.app.service.mono.context.properties.PropertyNames;
+import com.swirlds.config.api.ConfigProperty;
+
+/**
+ * Configuration for the NetworkAdmin service. Contains only the path to the upgrade artifacts directory.
+ *
+ * @param upgradeArtifactsPath path to the location where upgrade zip files are stored
+ */
+public record NetworkAdminServiceConfig(
+        @ConfigProperty(PropertyNames.UPGRADE_ARTIFACTS_PATH) String upgradeArtifactsPath) {}
```

Comment:
Default value is missing. Since we will deprecate / remove PropertyNames in future you should define the name of the property directly

Reply:
Fixed